python/votingapp/README.md

```diff
@@ -0,0 +1,106 @@
+# Voting App
+
+The votingapp demostrates how you can create a Fission function to connect to a relational database like Postgres and perform SQL operations. This is a basic app to help you understand how you can leverage Fission functions for working with databases.
```

Comment:
Please keep each complete sentence on a new line.

```
This is a sentence1.
This is a sentence2.
```
This helps in improving readability for markdown. Same for the rest of the document.

Reply:
fixed